### PR TITLE
JWT handling and Element+ UI

### LINKED ISSUE #7: Get and display attestation JWT when the user navigates to the upload-data page.
The JWT should be verified on the frontend, and the results of verification displayed. If verification fails, retry once, and if it still fails, an error page should be displayed to the user.

The relevant values that should be displayed to the user is:
  - x-ms-sgx-mrsigner: hex encoded value of the “mrsigner” field of the quote
  - x-ms-sgx-mrenclave: hex encoded value of the “mrenclave” field of the quote

The user should be able to copy the JWT on the file upload page to allow them to verify the token themselves if needed.

The JWT token should be stored inside of the session state of the application. It probably makes the most sense to store just the token payload.

### PR DESCRIPTION
**Pull request by jongbonga**

### CODE REVIEW

File: src/utils/jwt.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  if (headObj == null) {
+    throw new Error("Invalid Token");
+  }
+
+  const key = getVerifyKey((headObj as { kid: string }).kid);
+
+  const isValid = KJUR.jws.JWS.verifyJWT(token, key, {
+    iss: [ISSUER],
+    alg: ["RS256"],
+    gracePeriod: 20 * 20 * 20 * 100 * 100
```
Comment (longtomjr) [inline src/utils/jwt.ts:30]: We should add a flag to only have the grace period extended locally when developing.

**Review by longtomjr — APPROVED** [review]
Looks good! I am happy for us to merge this now and incorporate any feedback in later PRs.

File: src/utils/jwt.ts (hunk 1 above)
Comment (PiDelport) [reply src/utils/jwt.ts:30]: Is there any significance to the `20 * 20 * 20 * 100 * 100`?

(Was it supposed to be `60 * 60 * 24 * …`?)

**Review by PiDelport — APPROVED** [review]
Looks good!

I added some things I spotted directly, for the sake of time: can you give those a final check before merging?

File: src/utils/jwt.ts (hunk 1 above)
Comment (longtomjr) [reply src/utils/jwt.ts:30]: (nope, it is just a large number for testing with old tokens)

**Pull request merged**